base/libgit2/types.jl

```diff
@@ -174,7 +174,50 @@ function RemoteCallbacks(credentials::Ptr{Void}, payload::Ref{Nullable{AbstractC
     RemoteCallbacks(credentials=credentials_cb(), payload=pointer_from_objref(payload))
 end
 
-if LibGit2.version() >= v"0.24.0"
+immutable ProxyOptions
```

Comment:
should put the definition of this type under the `if LibGit2.version() >= v"0.25.0"` as well, it shouldn't be added to the module unless you're building in a non-default configuration and absolutely need it